### PR TITLE
gccounter module for hmmcopy

### LINKED ISSUE #1002: new module: hmmcopy/gcCounter
I think it would be good to have a module for the hmmcopy utility function gcCounter.

- [X ] This module does not exist yet with the [`nf-core modules list`](https://github.com/nf-core/tools#list-modules) command
- [X ] There is no [open pull request](https://github.com/nf-core/modules/pulls) for this module
- [X ] There is no [open issue](https://github.com/nf-core/modules/issues) for this module
- [X ] If I'm planning to work on this module, I added myself to the `Assignees` to facilitate tracking who is working on the module

### PR DESCRIPTION
**Pull request by SPPearce**
Closes #1002 by adding a new module hmmcopy/gccounter

- [X] This comment contains a description of changes (with reason).
- [X] If you've fixed a bug or added code that should be tested, add tests!
- [X] If you've added a new tool - have you followed the module conventions in the [contribution docs](https://github.com/nf-core/modules/tree/master/.github/CONTRIBUTING.md)
- [ ] If necessary, include test data in your PR.
- [X] Remove all TODO statements.
- [X] Emit the `versions.yml` file.
- [X] Follow the naming conventions.
- [X] Follow the parameters requirements.
- [X] Follow the input/output options guidelines.
- [X] Add a resource `label`
- [X] Use BioConda and BioContainers if possible to fulfil software requirements.
- Ensure that the test works with either Docker / Singularity. Conda CI tests can be quite flaky:
    - [ ] `PROFILE=docker pytest --tag <MODULE> --symlink --keep-workflow-wd`
    - [X] `PROFILE=singularity pytest --tag <MODULE> --symlink --keep-workflow-wd`
    - [ ] `PROFILE=conda pytest --tag <MODULE> --symlink --keep-workflow-wd`

### CODE REVIEW

File: modules/hmmcopy/gccounter/main.nf [hunk 1]
```diff
@@ -0,0 +1,40 @@
+// Import generic module functions
+include { initOptions; saveFiles; getSoftwareName; getProcessName } from './functions'
+
+params.options = [:]
+options        = initOptions(params.options)
+
+def VERSION = '0.99.0'
```
Comment (chris-cheshire) [inline modules/hmmcopy/gccounter/main.nf:7]: This version does not match the conda/container versions

File: modules/hmmcopy/gccounter/main.nf [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+    label 'process_low'
+    publishDir "${params.outdir}",
+        mode: params.publish_dir_mode,
+        saveAs: { filename -> saveFiles(filename:filename, options:params.options, publish_dir:getSoftwareName(task.process), meta:[:], publish_by_meta:[]) }
+
+    conda (params.enable_conda ? "bioconda::hmmcopy=0.1.1" : null)
+    if (workflow.containerEngine == 'singularity' && !params.singularity_pull_docker_container) {
+        container "https://depot.galaxyproject.org/singularity/hmmcopy:0.1.1--h2e03b76_5"
+    } else {
+        container "quay.io/biocontainers/hmmcopy:0.1.1--h2e03b76_5"
```
Comment (chris-cheshire) [inline modules/hmmcopy/gccounter/main.nf:19]: See comment above

File: modules/hmmcopy/gccounter/main.nf [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+    }
+
+    input:
+    path fasta
+
+    output:
+    path "*.gc.wig"          , emit: wig
+    path "versions.yml" , emit: versions
```
Comment (chris-cheshire) [inline modules/hmmcopy/gccounter/main.nf:27]:
```suggestion
    path "*.gc.wig"    , emit: wig
    path "versions.yml", emit: versions
```

**Review by chris-cheshire — CHANGES_REQUESTED** [review]

File: modules/hmmcopy/gccounter/main.nf (hunk 1 above)
Comment (SPPearce) [reply modules/hmmcopy/gccounter/main.nf:7]: Hmm, I could have sworn I had seen a 0.99 version number on the github link, but can't see it now. The tool itself doesn't have any version information that I can find. I'll just use the 0.1.1 now then.

**Review by chris-cheshire — APPROVED** [review]

**Pull request merged**